### PR TITLE
Small & Prechecked checkboxes

### PR DESCRIPTION
**Pull request by okolomiets**
_**Need help on styles to avoid duplicated rules.**_

Use case demo:

### CODE REVIEW

File: src/directives/checkbox/checkbox.partial.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="dd__checkbox-wrapper">
-	<div class="dd__checkbox" ng-class="{'checked': isChecked}" ng-click="toggle()">
+	<div class="dd__checkbox{{isSmall ? '--small': ''}}" ng-class="{'checked': isChecked}" ng-click="toggle()">
```
Comment (freeman14) [inline src/directives/checkbox/checkbox.partial.html]: use additional class for modificator dd__checkbox--small

File: src/styles/blocks/input.less [hunk 2]
```diff
@@ -69,7 +70,7 @@ form {
         -moz-user-select: text;
         -ms-user-select: text;
         user-select: text;
-        max-width: calc(~"100% - "@labelMargin);
```
Comment (freeman14) [inline src/styles/blocks/input.less:72]: Why you remove this?

Comment (freeman14) [reply src/styles/blocks/input.less:72]: This using for calculate checkbox with with label and use `text-overflow: elipsis` for label

File: stories/styleGuide/atoms/inputs.html [hunk 3]
```diff
@@ -59,11 +59,14 @@
         </div>
     </div>
 
-    <div class="subtitle">Checkbox</div>
+    <div class="subtitle">Checkboxes</div>
     <div class="block row">
-        <checkbox ng-model="value">
+        <checkbox ng-model="model1">
```
Comment (freeman14) [inline stories/styleGuide/atoms/inputs.html]: don't use variables like `model1` and `myVar1`

**Review by freeman14 — CHANGES_REQUESTED** [review]

File: src/directives/checkbox/checkbox.partial.html (hunk 1 above)
Comment (okolomiets) [reply src/directives/checkbox/checkbox.partial.html]: Fixed.

File: src/styles/blocks/input.less (hunk 2 above)
Comment (okolomiets) [reply src/styles/blocks/input.less:72]: Reverted.

**Review by freeman14 — APPROVED** [review]

**Pull request merged**